test/blackbox-tests/test-cases/github4821.t/run.t

```diff
@@ -0,0 +1,15 @@
+  $ touch foo.opam
+  $ cat > dune-project<<EOF
+  > (lang dune 2.0)
+  > EOF
+  $ cat > dune <<EOF
+  > (library (name foo) (public_name foo) (enabled_if false))
+  > EOF
+  $ export DUNE_BUILD_DIR=$(mktemp -d -t github4821XXXXXX);
```

Comment:
Why is setting `DUNE_BUILD_DIR` necessary? The tests automatically replace the test case directory with`TEST_CASE_ROOT`.

Reply:
I confess to having copied the test from a previous one!